core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/huggingFace/HuggingFaceSpamSMSDetectionOpDesc.scala

```diff
@@ -0,0 +1,72 @@
+package edu.uci.ics.texera.workflow.operators.huggingFace
+
+import com.fasterxml.jackson.annotation.{JsonProperty, JsonPropertyDescription}
+import edu.uci.ics.amber.engine.common.workflow.{InputPort, OutputPort}
+import edu.uci.ics.texera.workflow.common.metadata.annotations.AutofillAttributeName
+import edu.uci.ics.texera.workflow.common.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.texera.workflow.common.operators.PythonOperatorDescriptor
+import edu.uci.ics.texera.workflow.common.tuple.schema.{AttributeType, Schema}
+
+class HuggingFaceSpamSMSDetectionOpDesc extends PythonOperatorDescriptor {
+  @JsonProperty(value = "attribute", required = true)
+  @JsonPropertyDescription("column to perform spam detection on")
+  @AutofillAttributeName
+  var attribute: String = _
+
+  @JsonProperty(
+    value = "spam result attribute",
+    required = true,
+    defaultValue = "is_spam"
+  )
+  @JsonPropertyDescription("column name of whether spam or not")
+  var resultAttributeSpam: String = _
+
+  @JsonProperty(
+    value = "Neutral result attribute",
+    required = true,
+    defaultValue = "score"
+  )
+  @JsonPropertyDescription("column name of score for classification")
+  var resultAttributeScore: String = _
```

Comment:
is this "score" the same as "probability"? if so, please change it universally to `probability`.